lib/kafka/client.rb

```diff
@@ -694,6 +694,15 @@ def partitions_for(topic)
       @cluster.partitions_for(topic).count
     end
 
+    # Counts the number of replicas for a topic's partition
+    #
+    # @param topic [String]
+    # @return [Integer] the number of replica nodes for the topic's partition
+
```

Comment:
Can you remove the blank line? It'll screw up the docs I think.

Reply:
Done